test/kubemark/resources/hollow-node_template.json

```diff
@@ -145,27 +157,48 @@
 				},
 				{
 					"name": "hollow-node-problem-detector",
-					"image": "gcr.io/google_containers/node-problem-detector:v0.2",
+					"image": "gcr.io/google_containers/node-problem-detector:v0.3",
```

Comment:
Then I don't understand. If the image already starts npd, what is the below "command" doing? Is it starting yet another NPD? So effectively we have two NPDs started from this container?